plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSink.java

```diff
@@ -92,7 +90,7 @@
     private final List<DeltaLakeColumnHandle> dataColumnHandles;
     private final int[] dataColumnInputIndex;
     private final List<String> dataColumnNames;
-    private final List<Type> dataColumnTypes;
```

Comment:
unrelated change - pls remove from this commit.

Reply:
fixed